src/Microsoft.Health.Fhir.Core/AssemblyInfo.cs

```diff
@@ -9,3 +9,5 @@
 [assembly: NeutralResourcesLanguage("en-us")]
 [assembly: InternalsVisibleTo("Microsoft.Health.Fhir.R4.Api.UnitTests")]
 [assembly: InternalsVisibleTo("Microsoft.Health.Fhir.Stu3.Api.UnitTests")]
+[assembly: InternalsVisibleTo("Microsoft.Health.Fhir.Tests.E2E.R4")]
```

Comment:
I don't think the E2E assemblies should have dependency on Core assembly (I haven't seen what the exact dependencies are). I think the E2E tests should be performed purely through HTTP layer and shouldn't need to know the internals of the core implementation.

Reply:
The dependency this enables is for some constants used in the smart tests. 

Additionally, there is a dependency for `ResourceElement`, but that's publicly exposed in core.